libs/expo/shared/static/src/lib/regex.ts

```diff
@@ -3,4 +3,5 @@ export const Regex = {
   date: /^(0[1-9]|1[0-2])\/(0[1-9]|[12]\d|3[01])\/(19|20)\d{2}$/,
   time: /^(?:[01]\d|2[0-3]):[0-5]\d$/,
   email: /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/,
+  phoneNumber: /^\d{10}$/,
```

Comment:
```suggestion
  phoneNumber: /^[2-9]\d{2}[2-9]\d{6}$/,
```
I think this is closer to what we want. The first and fourth digit must be between 2 and 9, since neither an area code nor a 7 digit phone number can begin with a 0 or 1

Reply:
done